docker-compose.yml

```diff
@@ -53,12 +52,29 @@ services:
     volumes:
     # speed up development by mounting some local directories with hot reload
     - ./packages/worker/src:/app/packages/worker/src
-    - ./packages/worker/test:/app/packages/worker/test
     - ./packages/shared/build:/app/packages/shared/build
     env_file:
     - .env
     logging: *loki
 
+  # for running test commands
+  test: 
+    build: .
+    image: aragon/ipfs-background-service
+    command: sleep 365d
```

Comment:
isn't there something better to do to avoid docker stoping this container?

Reply:
Well there needs to be a running process, sleep works perfect